backend/Dockerfile

```diff
@@ -1,14 +1,22 @@
-FROM python:3.8-slim
+FROM python:3.10-slim
```

Comment:
Why not 3.11?

Reply:
oh, no reason at all :P we can turn it up to 11. I'll change that